CONTRIBUTING.md

```diff
@@ -1,39 +1,33 @@
 Contributing
 ============
 
-Thank you for considering contributing to Mastodon 🐘
+Thank you for considering contributing to Florence Mastodon
 
 You can contribute in the following ways:
 
 - Finding and reporting bugs
 - Translating the Mastodon interface into various languages
 - Contributing code to Mastodon by fixing bugs or implementing features
 - Improving the documentation
+- Joining the conversation on our [Chat]
+- Participate in OutReach or other activities
+- Proof-read
+- And much more
 
-If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
+If you want to contribute with monetary support you can do so via our [Open Collective](https://opencollective.com/florence-social) (which isn't publicly launched yet)
 
-## Bug reports
+## Adding an Issue for Bug Report or Feature Request
 
-Bug reports and feature suggestions can be submitted to [GitHub Issues](https://github.com/tootsuite/mastodon/issues). Please make sure that you are not submitting duplicates, and that a similar report or request has not already been resolved or rejected in the past using the search function. Please also use descriptive, concise titles.
+Bug reports and feature suggestions can be submitted to [GitHub Issues](https://github.com/florence-social/mastodon-fork/issues). Do not worry about submitting duplicates, but please make a cursory search to see if any similar reports or request have already been resolved or rejected in the past using the search function. If you find a bug or feature request which matches yours you can join the conversation by adding your commentary to that issue.
 
-## Translations
-
-You can submit translations via [Crowdin](https://crowdin.com/project/mastodon). They are periodically merged into the codebase.
-
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/mastodon/localized.svg)][crowdin]
-
-## Pull requests
+Try to describe your issue, the use-case, and additional considerations when submitting a feature request.
```

Comment:
"and any additional considerations" might be better. Omitting "any" implies people may need to ask what is permitted.